### PR TITLE
add site linkable behaviour

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: ApiBundle/Controller/ContentController.php [hunk 1]
```diff
@@ -113,7 +114,7 @@ public function listAction(Request $request)
         );
```
Comment (itkg-alavieille) [inline ApiBundle/Controller/ContentController.php]: for filter the contents according they are linked to a site, should update the mapping with 
'site_linked' => array('key' => 'siteLinked', 'type' => 'boolean'

File: ApiBundle/Controller/ContentController.php [hunk 2]
```diff
@@ -113,7 +114,7 @@ public function listAction(Request $request)
...
 
         $repository =  $this->get('open_orchestra_model.repository.content');
-        $contentCollection = $repository->findByContentTypeInLastVersionForPaginateAndSearch($contentType, $columnsNameToEntityAttribute, $columns, $search, $order, $skip, $limit);
+        $contentCollection = $repository->findByContentTypeInLastVersionForPaginateAndSearchAndSiteId($contentType, $columnsNameToEntityAttribute, $columns, $search, $siteId, $order, $skip, $limit);
```
Comment (itkg-ngilain) [inline ApiBundle/Controller/ContentController.php]: This naming is becoming totallly absurd :s
@itkg-alavieille dind'nt want to enumerate all the params so he suffixed it with 'ForPaginateAndSearch' instead. Now there's a random param added to the end ('AndSiteId'). Where is the consistency to talk about the SiteId and not about the other params ? (Without speaking of the ridiculous naming the method is going to have now :p)

Comment (nicolasThal) [inline ApiBundle/Controller/ContentController.php]: Yes, but contrary to all the search parameters, which are given by  the DataTable request, the siteId is an internal parameter, that's why I added it in the method name

File: ApiBundle/Controller/ContentController.php (hunk 1 above)
Comment (nicolasThal) [inline ApiBundle/Controller/ContentController.php]: discussed with @itkg-alavieille and this cannot be done

File: ApiBundle/Controller/ContentController.php (hunk 2 above)
Comment (itkg-ngilain) [inline ApiBundle/Controller/ContentController.php]: I don't make this a 'PR-stopper' as we should have a general brainstorming on the naming of the repo methods.

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**